src/main/java/com/endava/soa_5/exception_and_io/FileConverter.java

```diff
@@ -0,0 +1,90 @@
+package com.endava.soa_5.exception_and_io;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static java.util.regex.Pattern.compile;
+
+public class FileConverter {
+
+    private String outputFileName;
+    private final String FILE_NAME;
```

Comment:
I don't think that this is a constant, so you may change the style of declaring it (using uppercase).
Also it's related to other properties.

Reply:
Done!